docs/ota-client-guide/modules/ROOT/pages/release-process.adoc

```diff
@@ -24,7 +24,7 @@ This is normally a good time to update the link:https://github.com/advancedtelem
 There are a few additional documents that should be updated to refer to the new release:
 
 * The link:https://github.com/advancedtelematic/aktualizr/blob/master/docs/README.adoc#reference-documentation[docs README] contains a table with links to the doxygen docs for each release.
-* The xref:install-garage-sign-deploy.adoc[garage-deploy] installation guide contains a reference to the latest release of the garage-deploy Debian package.
+* Several pages contains references to the latest release of aktualizr and/or garage-deploy via a variable set in a xref:_partials/aktualizr-version.adoc[doc snippet] created for this purpose.
```

Comment:
```suggestion
* Several pages contains references to the latest release of aktualizr and/or garage-deploy via a variable set in a https://github.com/advancedtelematic/aktualizr/blob/master/docs/ota-client-guide/modules/ROOT/pages/_partials/aktualizr-version.adoc[doc snippet] created for this purpose.
```

This is a (rare) situation where the xref syntax isn't appropriate--on the docsite, that link won't work because of the way Antora deals with partials and xrefs.

Reply:
So we're stuck with a link to github even from the docs portal? Doesn't bother me, just confirming.

Reply:
Yes. It's also the conceptually correct thing to do, if you think about it--what would that link even point to, in the rendered version of the site? It renders as a completely empty file, because it's only some comment lines and an attribute declaration.